armi/reactor/tests/test_flags.py

```diff
@@ -104,6 +104,35 @@ def test_isPickleable(self):
         stream = pickle.dumps(flags.Flags.BOND | flags.Flags.A)
         flag = pickle.loads(stream)
         self.assertEqual(flag, flags.Flags.BOND | flags.Flags.A)
+        
+    def test_areHashable(self):
+        """
+        Test that the implemented ``Flags`` can be hashed. 
+        
+        This is important for inserting flags into the keys of a dictionary.
+        """
+        self.assertTrue(hash(flags.Flags.FUEL))
+        
+    def test_areSortable(self):
+        """Test that flags can be ordered."""
+        self.assertEqual(flags.Flags.FUEL, flags.Flags.FUEL)
+        #self.assertGreater(flags.Flags.CONTROL, flags.Flags.FUEL)
+        #self.assertGreaterEqual(flags.Flags.CONTROL, flags.Flags.FUEL)
+        #self.assertLessEqual(flags.Flags.FUEL, flags.Flags.CONTROL)
+        #self.assertLess(flags.Flags.FUEL, flags.Flags.CONTROL)
+        self.assertNotEqual(flags.Flags.FUEL, flags.Flags.CONTROL)
+        
+        strings = ["control", "axial shield", "radial shield", "fuel", "structure",
+                   "plenum", "duct"]
+        sortedStrings = ["axial shield", "control", "duct", "fuel", "plenum", "radial shield",
+                         "structure"]
+        self.assertEqual(sorted(strings), sortedStrings)
+        
+        print(sorted([flags.Flags.fromString(x) for x in strings]))
+        print([flags.Flags.fromString(x) for x in sortedStrings])
+        self.assertEqual(sorted([flags.Flags.fromString(x) for x in strings]), 
+                         [flags.Flags.fromString(x) for x in sortedStrings])
+        
```

Comment:
WIP - don't comment. Will fix.

Reply:
Any notes on progress here? 

Just curious, I don't mean to rush.

Reply:
I think that Nick just recommended that I implement `__eq__` and get rid of the others. Sorry my comment should have been much better... 

I'm still held up in the core below but haven't had time to work on it. This is an important but fix though to the converter.

Reply:
Also I think I implemented hash on the strings from the flag but I think that flags should be hashable without that.